testing/test_aux.out

```diff
@@ -5,14 +5,33 @@ Mudacl exit code 0
 Validator exit code 0
 Lint checks
 cmd/inbuild exit code 0
+Running bacext test...
+RESULT pass protocol.bacnet.version
+****************************** START ******************************
```

Comment:
remove *** Start *** end.  It's too much deatil for this.  Look at how everything else is done... minimum subset of information necessary. This is not a "user-friendly" file.